python/src/filesystem.rs

```diff
@@ -229,7 +229,7 @@ impl DeltaFileSystemHandler {
         let file = self
             .rt
             .block_on(ObjectInputFile::try_new(
-                self.rt.clone(),
+                Arc::clone(&self.rt),
```

Comment:
Just out of curiosity, I would have thought the statements are equivalent, since rt is inside an Arc ... ?